### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by mathiasrw** [slevithan/regex#17]
Amazing thing you have created here. Thank you for creating this for the world to enjoy. 

(you can close this issue now that you read this.)

**Comment by slevithan** [comment]
Thanks, @mathiasrw! That means a lot since I've put in a *lot* of time and sleepless nights trying to make this project as good as it can be. Super glad you find it useful!

**Comment by mathiasrw** [comment]
It’s the drive of obsession that gives us the joy to keep going, creating, and refining. The idea must manifest. I know the feeling.

Thanks for the input on RexReplace. It was while researching for that library that I discovered this one.

> Super glad you find it useful!

Actually, I haven’t used it yet - I only found it yesterday! I need to really dig in and understand the implication of all the new possibilities in regex I now see. Recently, I made a small parser for a data structure using plain regex, and I have a feeling these new opportunities could have made things much easier. Im wondering if there is the opportunity to automate making  grammar / lexical analysis where regex is handling much of the heavy lifting (I feel new ideas are forming. I should probably buy some new domain names and let them sit alongside all my other ideas...). 

On a more practical note, I’m also very inspired by how you use backtick syntax to mix data with static information. It’s a great idea to consider how things can become more expressive by blending hardcoded data with object references. It adds a new level of depth to engaging with the library for developers.

And I love that you created a Babel plugin. I’ve been thinking of doing something similar for [alasql](https://github.com/AlaSQL/alasql) to precompile SQL into JavaScript functions instead of doing it on the fly. I just didn’t have a good example of all the pieces coming together to start that journey. Now I do. 

Cheers,  
Mathias
